.github/workflows/container-validation-backends.yml

```
@@ -212,12 +291,52 @@ jobs:
+          name: build-metrics-vllm
+          path: build-metrics/
+        continue-on-error: true  # Don't fail if artifact doesn't exist
+
       - name: Upload Complete Workflow Metrics
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           WORKFLOW_INDEX: ${{ secrets.WORKFLOW_INDEX }}
           JOB_INDEX: ${{ secrets.JOB_INDEX }}
           STEPS_INDEX: ${{ secrets.STEPS_INDEX }}
+          # Pass build metrics as environment variables
+          BUILD_DURATION_SEC: ${{ needs.vllm.outputs.build-duration-sec }}
```

actionlint (1.7.7)

property "vllm" is not defined in object type {backend-status-check: {outputs: {}; result: string}}

(expression)